File: 231213/이상한 진수 2/awkward-digits-2.js
const fs = require('fs')
const inputAll = () => fs.readFileSync('/dev/stdin').toString().split('\n');
const print = (...args) => console.log(...args);

const [binStr] = inputAll();
const binArr = Array.from(binStr);

for(let i = 0; i < binArr.length; i++){
    const binChar = binArr[i];

    if(binChar === '0'){
        binArr[i] = '1';

        print(parseInt(binArr.join(''), 2));
        return;
    }
}

binArr[binArr.length - 1] = '0'
print(parseInt(binArr.join(''), 2));

File: 231213/이상한 진수 2/awkward-digits-2.py
binStr = list(input())

# 앞에서부터 탐색해서 0이 보이면 그걸 바꾼다
# 0이 없다면 가장 뒤에 것을 바꾼다.

for i in range(len(binStr)):
    if binStr[i] == '0':
        binStr[i] = '1'
        print(int(''.join(binStr), 2))
        exit(0)

binStr[len(binStr) - 1] = '0'
print(int(''.join(binStr), 2))